### PR TITLE
add test for IHC slides and tiles

### PR DESCRIPTION
**Pull request by nicolebussola**

### CODE REVIEW

File: tests/integration/test_tiler.py [hunk 1]
```diff
@@ -117,55 +150,88 @@ def test_extract_tiles_respecting_the_given_tile_size(
 
 class DescribeGridTiler:
     @pytest.mark.parametrize(
-        "fixture_slide, level,check_tissue, expectation",
+        "fixture_slide, binary_mask, tile_size, level,check_tissue, expectation",
```
Comment (alessiamarcolini) [inline tests/integration/test_tiler.py:153]: why here all the tile sizes are squared?

File: tests/integration/test_tiler.py [hunk 2]
```diff
@@ -174,59 +240,58 @@ def it_locates_tiles_on_the_slide(
 
 class DescribeScoreTiler:
     @pytest.mark.parametrize(
-        "fixture_slide, level, check_tissue, expectation",
+        "fixture_slide,tile_size, level,check_tissue, expectation",
```
Comment (alessiamarcolini) [inline tests/integration/test_tiler.py:243]: same here

File: tests/integration/test_tiler.py [hunk 3]
```diff
@@ -244,7 +309,7 @@ def it_locates_tiles_on_the_slide(
             # no binary mask object passed to locate_tiles
             # default value = BiggestTissueBoxMask
             tiles_location_img = scored_tiles_extractor.locate_tiles(
-                slide, scale_factor=10
+                slide, scale_factor=8
```
Comment (alessiamarcolini) [inline tests/integration/test_tiler.py:312]: why change to 8?

Comment (nicolebussola) [reply tests/integration/test_tiler.py:312]: no real reason, only the ihc slide is really small

File: tests/integration/test_tiler.py (hunk 1 above)
Comment (nicolebussola) [reply tests/integration/test_tiler.py:153]: I can change that, but is it important?

Comment (alessiamarcolini) [reply tests/integration/test_tiler.py:153]: it is probably important as it is with the random tiler

Comment (nicolebussola) [reply tests/integration/test_tiler.py:153]: kkkk

File: tests/integration/test_tiler.py [hunk 4]
```diff
@@ -279,7 +279,7 @@ class DescribeScoreTiler:
             ),
         ],
     )
-    def it_locates_tiles_on_the_slide(
+    def it_locates_tiles_on_the_slide22(
```
Comment (alessiamarcolini) [inline tests/integration/test_tiler.py:282]: title 👀

Comment (nicolebussola) [reply tests/integration/test_tiler.py:282]: fucccck

**Review by alessiamarcolini — APPROVED** [review]

**Review by ernestoarbitrio — APPROVED** [review]

**Pull request merged**